File: app/src/main/java/ru/michanic/mymot/Models/Volume.java
package ru.michanic.mymot.Models;

import io.realm.RealmList;
import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class Volume extends RealmObject {

    @PrimaryKey
    private int id;

    private String code;
    private String image;
    private String name;
    private int sort;
    private int min;
    private int max;

    // связи
    private RealmList<Model> models;

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getImage() {
        return image;
    }

    public RealmList<Model> getModels() {
        return models;
    }

    public String getCode() {
        return code;
    }
}

File: app/src/main/java/ru/michanic/mymot/Kotlin/Models/Volume.kt
package ru.michanic.mymot.Kotlin.Models

import io.realm.RealmList
import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import ru.michanic.mymot.Kotlin.Models.Model

open class Volume : RealmObject() {
    @PrimaryKey
    var id = 0
    var code: String? = null
    var image: String? = null
    var name: String? = null
    private var sort = 0
    var min = 0
    var max = 0

    var models: RealmList<Model>? = null
}